draft-ietf-dnsop-terminology-bis-latest.xml

```diff
@@ -168,9 +168,9 @@ An ordered list of one or more labels.</t>
 also applies to systems other than the DNS. <xref target="RFC1034"/> defines the "domain
 name space" using mathematical trees and their nodes in graph theory, and the definition
 in <xref target="RFC1034"/> has the same practical result as the definition here. Using
-graph theory, a domain name is a list of labels identifying a portion along one edge of an
-acyclic directed graph. A domain name can be relative to other parts of the tree, or it
-can be fully qualified (in which case, it ends at the common root of the graph).</t>
+graph theory, a domain name is a list of labels identifying a portion along one edge of a
+directed acyclic graph. A domain name can be relative to other parts of the tree, or it
```

Comment:
The new term dominates the old (cf. [Google Trends](https://trends.google.com/trends/explore?q=%22acyclic%20directed%20graph%22,%22directed%20acyclic%20graph%22)).